### ISSUE TITLE
Mysql PK Column Remark removes AUTO_INCREMENT

### ISSUE THREAD

**Issue by vgropp** [liquibase/liquibase#2301]
## Environment

**Liquibase Version**: 4.6.2

**Liquibase Integration & Version**: spring boot

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**: 5.7.mysql_aurora.2.08.2

**Operating System Type & Version**: unknown

## Description

Since 4.6.2 adding a column remark to a PK column with Auto_increment removes the auto increment from the column. For me, It looks like it was introduced by https://github.com/liquibase/liquibase/commit/695c3ab79fab9ba1819b94967faa21613c9606c0. Other Attributes, ie. NOT NULL is still in place.

## Steps To Reproduce

```
databaseChangeLog:
  - changeSet:
      comment: add a table
      id: TICKET-1
      author: vg
      changes:
        - createTable:
            tableName: t_table
            columns:
              - column:
                  name: id
                  remarks: PK
                  type: bigint
                  autoIncrement: true
                  constraints:
                    primaryKey: true
                    nullable: false
```

## Actual Behavior

This results in:
```
 10:50:26.396 [Test worker] DEBUG liquibase.executor.log MDC=[:::] - CREATE TABLE `someport/test`.t_table (id BIGINT AUTO_INCREMENT NOT NULL, CONSTRAINT PK_T_TABLE PRIMARY KEY (id)) 
 10:50:26.414 [Test worker] DEBUG liquibase.executor.log MDC=[:::] - 0 row(s) affected 
 10:50:26.414 [Test worker] DEBUG liquibase.database.log MDC=[:::] - Executing Statement: liquibase.statement.core.SetColumnRemarksStatement@56b0789a 
 10:50:26.415 [Test worker] DEBUG liquibase.executor.log MDC=[:::] - Executing with the 'jdbc' executor 
 10:50:26.415 [Test worker] DEBUG liquibase.executor.log MDC=[:::] - ALTER TABLE `someport/test`.t_table MODIFY COLUMN id BIGINT COMMENT 'PK' 
 10:50:26.435 [Test worker] DEBUG liquibase.executor.log MDC=[:::] - 0 row(s) affected 
```

```
CREATE TABLE `t_table ` (
  `id` bigint(20) NOT NULL COMMENT 'PK',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

## Expected/Desired Behavior

Expected is:
```
CREATE TABLE `t_table ` (
  `id` bigint(20) NOT NULL AUTO_INCREMENT COMMENT 'PK',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

Either it should add the auto_increment aswell or revert adding the remark (we actually don't care this much). Its hard to tell if there are other attributes of the PK wich will be removed.

**Comment by kataggart** [comment]
Thanks @vgropp for the great detail and research you provided here! We are taking a look at it.

Semi-related, wanted to make sure you knew about [https://forum.liquibase.org/]( https://forum.liquibase.org/) - given your experience working with Liquibase, we would love to see you over there. It's becoming a solid resource for questions/answers/suggestions/etc.

**Comment by vgropp** [comment]
Hi, thanks for the hint to the forum and happy new year

If someone can give me a hint on how to solve this, ie just add auto_increment support to mysql column remarks or maybe disable remarks on PK, i can help to fix it as well.

**Comment by nvoxland** [comment]
I created a fix for this in #2752 and liquibase/liquibase-test-harness#211
